### PR TITLE
Disable failing tests in CI

### PR DESCRIPTION
**Pull request by meg-gupta**

### CODE REVIEW

**Comment by meg-gupta** [comment]
@swift-ci smoke test and merge

**Review by hyp — APPROVED** [review]
Thanks!

**Comment by meg-gupta** [comment]
@swift-ci smoke test

File: test/IRGen/typed_throws.swift [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 // RUN: %target-swift-frontend -primary-file %s -emit-ir -enable-experimental-feature TypedThrows -disable-availability-checking -runtime-compatibility-version 5.8 -disable-concrete-type-metadata-mangled-name-accessors | %FileCheck %s --check-prefix=CHECK-NOMANGLE
 
+// REQUIRES: rdar117673132
```
Comment (aschwaighofer) [inline test/IRGen/typed_throws.swift:5]: Does this test only fail on arm64e?
Can we XFAIL: CPU=arm64e instead?

**Comment by meg-gupta** [comment]
@swift-ci smoke test

**Pull request merged**